README.rst

```diff
@@ -49,6 +49,8 @@ The user should provide a YAML configuration file defining the operator:
 -  polymorphic type attributes.
 -  other attributes.
 -  documentation.
+-  devices compatibilities.
+-  optional external libraries.
```

Comment:
`device compatibilities` to `devices`
`optional external libraries` to `makefile` options